Kentor.AuthServices/SAML2P/Saml2Response.cs

```diff
@@ -505,12 +508,19 @@ private IEnumerable<ClaimsIdentity> CreateClaims(IOptions options)
                     var token = (Saml2SecurityToken)handler.ReadToken(reader);
                     handler.DetectReplayedToken(token);
 
+                    // Have checked for replayed token, don't need to validate OneTimeUse (Saml2SecurityTokenHandler doesn't support it either)
+                    if (token.Assertion.Conditions != null)
```

Comment:
The if statement is not needed, at least not with the existing tests. Try adding a test with a saml2 response without conditions and see if it makes the if statement needed. I think it's blocked somewhere earlier as the replay detection requires a NotOnOrAfter.